myriad-scheduler/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/MyriadFileSystemRMStateStore.java

```diff
@@ -58,6 +58,8 @@ public synchronized void initInternal(Configuration conf) throws Exception {
       //something is broken
       throw new RuntimeException("Could not find valid updateFile Method");
     }
+    
+    startInternal();
```

Comment:
Okay, I've had about 5 minutes to look at this today. :) I added the @VisibleForTesting annotation to the MyriadFileSystemRMStateStore startInternal implementation and now the JUnit tests pass. Is everyone cool with this solution?

--John

Sent from my iPhone

> On Jun 9, 2016, at 10:33 PM, Darin <notifications@github.com> wrote:
> 
> In myriad-scheduler/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/MyriadFileSystemRMStateStore.java:
> 
> > @@ -58,6 +58,8 @@ public synchronized void initInternal(Configuration conf) throws Exception {
> >        //something is broken
> >        throw new RuntimeException("Could not find valid updateFile Method");
> >      }
> > +    
> > +    startInternal();
> Why was this added?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>